lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -71,30 +78,49 @@ data CabalPackageInfo = CabalPackageInfo
 preprocessorIdentifier :: String
 preprocessorIdentifier = "__preprocessor-apply-packages"
 
-run :: MonadObelisk m => m ()
+profile
+  :: MonadObelisk m
+  => String
+  -> [String]
+  -> m ()
+profile profileBasePattern rtsFlags = withProjectRoot "." $ \root -> do
+  putLog Debug "Using profiled build of project."
+
+  time <- liftIO getCurrentTime
+  let profileBaseName = formatTime defaultTimeLocale profileBasePattern time
+
+  putLog Debug $ T.pack $ "Storing profiled data under base name of " <> profileBaseName
+
+  liftIO $ createDirectoryIfMissing False $ takeDirectory profileBaseName
```

Comment:
This should be `True` to create parents too. That way patterns like `profile/%Y-%m-%dT%H:%M:%S/run` will work.